queries.sql

```diff
@@ -7,4 +7,69 @@ SELECT date_of_birth FROM animals  WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 't';
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN TRANSACTION;
+SAVEPOINT SP1;
+UPDATE animals
+SET species = 'undefined';
+ROLLBACK TO SP1;
+
+BEGIN TRANSACTION;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon%';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
+BEGIN TRANSACTION;
+
+SAVEPOINT SP2;
+
+DELETE FROM animals;
+
+ROLLBACK
+
+BEGIN TRANSACTION;
+
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT SP3;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO SP3;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT MAX(escape_attempts) FROM animals;
+
+SELECT MIN(weight_kg) FROM animals  WHERE species = 'digimon';
+
+SELECT MIN(weight_kg) FROM animals WHERE species = 'pokemon';
+
+SELECT MAX(weight_kg) FROM animals WHERE species = 'digimon';
+
+SELECT MAX(weight_kg) FROM animals WHERE species = 'pokemon';
+
```

Comment:
- [ ] It would be nice if you use Group by here instead of making 4 queries just make one query because in the requirements you are directed to make only one query for minimum and maximum weights_kg for each type of animal 👍
- [ ] It would be nice if after making all the changes you would take new screenshots of the new results and replace them with the older screenshots in the pull request summary 😊